Feature/json schema form tests

saulipurhonen opened the pull request:
### Description

Jest tests for form rendering from custom schema.

### Related issues

Closes #54 

### Type of change

<!-- Replace [ ] with [x] to select options. -->
<!-- Please delete options that are not relevant. -->

- [x] New feature (non-breaking change which adds functionality)

### Changes Made

- Created custom schema with different field types
- Test form rendering

### Testing

<!-- Are any tests part of this PR. -->
<!-- Replace [ ] with [x] to select options. -->
<!-- Please delete options that are not relevant. -->

- [x] Integration Tests

hannyle reviewed (approved):
Looks nice to me! 

I'm not sure if we have covered all the cases in forms yet but at least I have seen most of them. 
I don't see the test case for `enum` in `jest` test for example, or is it already covered in cypress test (?)

And I think it's good also to have some comments (in `jest` test) about which case/form as an example to easier to catch up later.

saulipurhonen commented:
> I'm not sure if we have covered all the cases in forms yet but at least I have seen most of them.

There can be some underlying special cases but I think we can add those later if found.

> I don't see the test case for `enum` in `jest` test for example, or is it already covered in cypress test (?)

Good call, missed that one entirely. Introduced test for `enum` field just now.
 
> And I think it's good also to have some comments (in `jest` test) about which case/form as an example to be easier to catch up later.

Very good point. Added some comments to latest commit.

hannyle reviewed (approved):
I think we have covered most of the cases for now. 
If later any specific cases still need to be added, we could have them in another PR.
